src/modules/mavlink/mavlink_receiver.h

```diff
@@ -203,6 +207,8 @@ class MavlinkReceiver
 	orb_advert_t _time_offset_pub;
 	orb_advert_t _follow_target_pub;
 	orb_advert_t _transponder_report_pub;
+	std::array<orb_advert_t, 4> _gps_inject_data_pub;
```

Comment:
I'd like to avoid the std lib dependency - can you convert this into a normal C array instead?